### DISCUSSION TITLE
Read Speed with the IBT Class

### DISCUSSION THREAD

**Discussion by wlt224** [kutu/pyirsdk#40] [General]
Hello, sorry if this is a silly question.
Python is new to me, and I'm just an engineer trying to perform some data analysis, not a programmer.

I'm attempting to extract data, such as speed, from an IBT file. According to the example in the tutorial, this should work. Reading the CarSetup, as mentioned in a previous question, works fine, but unfortunately, the speed extraction does not seem to be functioning: IndexError: list index out of range

<img width="142" alt="image" src="https://github.com/kutu/pyirsdk/assets/123174853/aeff16fa-0192-456f-b38a-d2c3006ebb81">

 Thank you for your great work!

**Comment by kutu** [comment]
```py
ibt = irsdk.IBT()
ibt.open(file_path)
print(ibt.get(0, 'Speed'))
print(ibt.get_all('Speed'))
ibt.close()
```

**Comment by wlt224** [comment]
Awesome, thanks!
